token/program-2022/src/extension/metadata_pointer/mod.rs

```diff
@@ -0,0 +1,26 @@
+use {
+    crate::{
+        extension::{Extension, ExtensionType},
+        pod::OptionalNonZeroPubkey,
+    },
+    bytemuck::{Pod, Zeroable},
+};
+
+/// Instructions for the MetadataPointer extension
+pub mod instruction;
+/// Instruction processor for the MetadataPointer extension
+pub mod processor;
+
+/// Metadata pointer extension data for mints.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Pod, Zeroable)]
+pub struct MetadataPointer {
+    /// Authority that can set the metadata address
+    pub authority: OptionalNonZeroPubkey,
+    /// Account address that holds the metadata
+    pub metadata_address: OptionalNonZeroPubkey,
```

Comment:
Assuming this is optional so you can nullify it later?
Would that mean if you initialize a metadata pointer and then null it later, you're still stuck with the 65 bytes?